gluecodium/src/main/resources/templates/swift/SwiftLambdaComment.mustache

```diff
@@ -0,0 +1,33 @@
+{{!!
+  !
+  ! Copyright (C) 2016-2023 HERE Europe B.V.
```

Comment:
```suggestion
  ! Copyright (C) 2023 HERE Europe B.V.
```

Reply:
This string is generated using `./gradlew :gluecodium:spotlessApply`, `build` will fail if we do not apply the spotless check, see - https://github.com/heremaps/gluecodium/blob/master/config/spotless/here_mustache.license#L3